README.md

```diff
@@ -62,14 +62,13 @@ If working on an ES5 project, extend the ES5 version of the configuration:
 }
 ```
 
-You can also add some "augmenting" configs on top of the "core" config by extending an array of linting configs. For example, this package provides a lodash linting config, which can be added to the ESNext config with the following configuration file:
+You can also add some "augmenting" configs on top of the "core" config by extending an array of linting configs. For example, this package provides a react linting config, which can be added to the ESNext config with the following configuration file:
 
 ```json
 {
   "extends": [
     "plugin:shopify/esnext",
-    "plugin:shopify/lodash",
-    "plugin:shopify/mocha"
+    "plugin:shopify/react",
```

Comment:
`react` extends from `esnext` - so we consider `react` a core config rather than an augmenting one. `jest` might be a better example of an augmenting config.